app/src/main/java/ar/scacchipa/twittercloneapp/di/SharedPrefs.kt

```diff
@@ -0,0 +1,11 @@
+package ar.scacchipa.twittercloneapp.di
+
+import android.app.Application
+import android.content.Context
+import android.content.SharedPreferences
+import ar.scacchipa.twittercloneapp.utils.Constants
+
+fun getSharedPrefs(androidApplication: Application): SharedPreferences {
```

Comment:
keep the same line of naming, if some provide methods start with get, use get eveywhere , if use provide prefix for nameing, keep using provide.